configs/riscv64-xs-ref_defconfig

```diff
@@ -17,6 +17,7 @@ CONFIG_CLINT_MMIO=0x38000000
 # CONFIG_MULTICORE_DIFF is not set
 CONFIG_RVB=y
 CONFIG_RVK=y
+CONFIG_RVH=y
```

Comment:
Please set default config value to false. Others will not use h-ext recently.

Reply:
ok